src/utils/constants.ts

```diff
@@ -4,3 +4,11 @@ export const apiUrls = {
   sandbox: 'https://api-sandbox.nofrixion.com/api/v1',
   production: 'https://api.nofrixion.com/api/v1',
 };
+
+export const dateRanges = {
+  today: 'Today',
+  yesterday: 'Yesterday',
+  last7Days: 'Last 7 Days',
+  last30Days: 'Last 30 Days',
+  last90Days: 'Last 90 Days',
+};
```

Comment:
This logic could be moved all together to use `date-fns` helper functions and this will make easier the behaviour of showing "Today" when someone picked today in the calendar (as the current behaviour is showing "Custom").